src/api.js

```diff
@@ -32,3 +32,16 @@ export const getArticle = (article_id) => {
       throw error;
     });
 };
+
+export const getComments = (article_id) => {
+  return NewsApi.get(`/articles/${article_id}/comments`)
+    .then(({ data: { comments } }) => {
+      console.log(comments);
+      console.log(typeof comments);
+      return comments;
+    })
+    .catch((error) => {
```

Comment:
This catch will stop you effectively error handling later, you're better off removing it.